src/ProjectTemplates/Web.Client.ItemTemplates/content/Scss/.template.config/template.json

```diff
@@ -6,7 +6,7 @@
   "generatorVersions": "[1.0.0.0-*)",
   "description": "SCSS is a language that compiles into CSS",
   "groupIdentity": "Microsoft.DotNet.Web.ClientItems.Scss",
-  "precedence": "100",
```

Comment:
Why is this going 1->7 as opposed to 6->7?

Reply:
Likely this is because this will be the first time we're updating the precedence value for this template, and 100 happened to be the initial value chosen. But since we recently updated the "identity" value, the precedence value should get updated as well to avoid the error mentioned in my PR comment.

https://github.com/dotnet/aspnetcore/pull/34598 made a similar change made for the other templates in 6.0.

Reply:
In other words, #34598 didn't change these templates, but #44120 did, hence the disparity.